docs/extensions_preview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Extensions Preview
+layout: docs
+---
+
+Introducing Tilt Extensions! Extensions are a simple way to bundle up Tiltfile functionality that should be shared with the world.
+
+We think that there are lots of higher level patterns that people are implementing with Tilt that could benefit from sharing, socialization and standardization. For example many users have implemented their own ways to force all kubernetes resources in to a namespace, inject sidecars or run tests. Extensions provide a way for users to package up these ideas and share them with the community.
```

Comment:
maybe lower level than you're looking for, but I think there's a good opportunity here to brag about our userbase and make this more positive, along the lines of "lots of our users have implement useful Tiltfile functionality that'd be great for others to be able to use as well" (we have lots of users! and they're writing great stuff!), while "many users have implemented their own ways" feels more eeyore-y "ugh our users have all this duplicate code"

Reply:
Love this feedback, thanks